.github/workflows/trivy.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
         with:
           scan-type: 'fs'
           scan-ref: .
```

Comment:
This `scan-ref` block is likely causing the false failures. Normally, we would isolate this to an operations directory, but we don't have that luxury here. Consider implementing `skip-dirs` to exclude the `.cache` directory: https://trivy.dev/v0.56/docs/configuration/skipping/